src/main/kotlin/org/arend/actions/ArendSmartSelectProvider.kt

```diff
@@ -20,6 +20,8 @@ class ArendSmartSelectProvider : SmartSelectProvider<ArendSmartSelectProvider.Co
 
     data class Context(val file: ArendFile, val editor: Editor, val selectionRange: TextRange)
 
+    override fun canIncreaseSelection(source: Context) = true
```

Comment:
There is no such method in the base class.

Reply:
It's 2022.1